docs/submit/osg-flock.md

```diff
@@ -38,32 +38,22 @@ and configure it only to forward jobs to the OSG.
 In other words, you do not need a whole HTCondor batch system just to have a local OSG access point.
 
 
-Before Starting
----------------
-Because jobs running in OSG run in a completely different environment than your local cluster,
-and require additional considerations for data movement,
-you or your campus will need to provide additional support for your users to enable them to
-properly use OSG.
-Send email to <help@opensciencegrid.org> saying that you would like to flock jobs to OSG,
-and we will [consult](#scheduling-a-planning-consultation) with you about the proper solution for your site and what
-needs to be set up at both the OSG and at your site.
+System Requirements
+-------------------
+The hardware requirement for an OSG access point depends on several factors such as number of users, 
+number of jobs and for example how I/O intensive those jobs are. Our minimum recommended configuration
+is 6 cores, 12 GB RAM and 1 TB of local disk. The hardware can be bare metal or virtual machine, but
+we do not recommend containers as these submit host are running a bunch of system services which can
+be difficult to configure in a container.
 
-Also consider the following technical requirements:
+Also consider the following configuration requirements:
 
 * __Operating system:__ A RHEL 7 compatible operating system.
 * __User IDs:__ If it does not exist already, the installation will create the Linux user ID `condor`.
-* __Host certificate:__ Recommended for authentication to OSG infrastructure.
-  See our [documentation](../security/host-certs.md) for instructions on how to request and install host certificates.
 * __Network:__ 
     * Inbound TCP port 9618 must be open.
     * The access point must have a public IP address with both forward and reverse DNS configured.
 
-As with all OSG software installations, there are some one-time steps to prepare in advance:
-
-* Obtain root access to the host
-* Prepare [the required Yum repositories](../common/yum.md)
-* Install [CA certificates](../common/ca.md)
-
```

Comment:
I'd prefer to keep the first two bullets here. For the third bullet, is `osg-flock` configured to use system CAs and do we not need to worry about IGTF CAs?

Reply:
We do not need CAs anymore - these are just straight tokens. The Yum repos install is in the next paragraph.

Reply:
IIRC there was one last place that GSI was used in the GlideinWMS inf. Was that between factory and FE?

Reply:
That is only FE->factory->glidein. No GSI needed between submit hosts and CM.